openpose/src/pose_estimation.py

```diff
@@ -29,12 +32,15 @@ def perform_openpose_pose_estimation(input_path: str, options: dict):
         op_wrapper.emplaceAndPop(op.VectorDatum([datum]))
 
         if datum.poseKeypoints is not None and len(datum.poseKeypoints) > 0:
-            pose_data.append({
-                'pose_keypoints': datum.poseKeypoints[0],
-                'face_keypoints': datum.faceKeypoints[0] if datum.faceKeypoints is not None and len(datum.faceKeypoints) > 0 else None,
-                'left_hand_keypoints': datum.handKeypoints[0][0] if datum.handKeypoints is not None and len(datum.handKeypoints) > 0 and datum.handKeypoints[0] is not None and len(datum.handKeypoints[0]) > 0 else None,
-                'right_hand_keypoints': datum.handKeypoints[1][0] if datum.handKeypoints is not None and len(datum.handKeypoints) > 1 and datum.handKeypoints[1] is not None and len(datum.handKeypoints[1]) > 0 else None,
-            })
+            if multi_person_detection:
+                pose_data.append(datum.poseKeypoints) # we are only interested in pose_keypoints for all people in the frame
+            else: # default behaviour for maskanyone
+                pose_data.append({
+                    'pose_keypoints': datum.poseKeypoints[0],
+                    'face_keypoints': datum.faceKeypoints[0] if datum.faceKeypoints is not None and len(datum.faceKeypoints) > 0 else None,
+                    'left_hand_keypoints': datum.handKeypoints[0][0] if datum.handKeypoints is not None and len(datum.handKeypoints) > 0 and datum.handKeypoints[0] is not None and len(datum.handKeypoints[0]) > 0 else None,
+                    'right_hand_keypoints': datum.handKeypoints[1][0] if datum.handKeypoints is not None and len(datum.handKeypoints) > 1 and datum.handKeypoints[1] is not None and len(datum.handKeypoints[1]) > 0 else None,
+                })
```

Comment:
Please have the same output format for multi_person_detection as without. So having the keys "poseKeypoints", "face_keypoints", and so on and setting "face...", "left..." and "right"... appropriately, like below.
This makes it consistent, does not introduce another edge case that we need to keep track of and also makes it maintainable.